examples/interaction/js_callbacks/setvalue.py

```diff
@@ -1,3 +1,14 @@
+''' An visualization of buttons in bokeh.models. This example demonstrates
+changing the value of an object when a certain event(like clicking of a button)
+occurs through SetValue.
```

Comment:
I think I know what you intended here, but it almost reads backward, i.e. that the SetValue caused the event. I would just replace this last bit since the "changing the value" was already mentioned above. 
```suggestion
happens.
```